### PR TITLE
Submission Table setup

### PR DESCRIPTION
**Pull request by djnunez-aot**
-Create submission table

image

### CODE REVIEW

File: submit-web/src/routes/_authenticated/_dashboard/projects/index.tsx [hunk 1]
```diff
@@ -1,10 +1,236 @@
...
+                variant="body1"
+                sx={{
+                  fontWeight: "bold",
+                  backgroundColor: BCDesignTokens.themeGold10,
+                }}
+              >
+                Active Submissions
+              </Typography>
+              <CardInnerBox sx={{ height: "100%" }}>
+                <TableContainer component={Box} sx={{ height: "100%" }}>
```
Comment (jadmnsaad) [inline submit-web/src/routes/_authenticated/_dashboard/projects/index.tsx]: why not all of this Table content is in the SubmissionTable component

File: submit-web/src/routes/_authenticated/_dashboard/projects/index.tsx [hunk 2]
```diff
@@ -1,10 +1,236 @@
...
+});
 
 export const Route = createFileRoute("/_authenticated/_dashboard/projects/")({
   component: ProjectsPage,
   meta: () => [{ title: "Projects" }],
 });
 
 function ProjectsPage() {
-  return <div>Projects Page</div>;
+  const plans: Array<Plan> = [
```
Comment (jadmnsaad) [inline submit-web/src/routes/_authenticated/_dashboard/projects/index.tsx]: please move this into a constants file, also setup a tanstack query api call that just returns this instead of making an actual call

File: submit-web/src/routes/_authenticated/_dashboard/projects/index.tsx (hunk 1 above)
Comment (djnunez-aot) [reply submit-web/src/routes/_authenticated/_dashboard/projects/index.tsx]: because the tables are sharing the same headers in the design

File: submit-web/src/routes/_authenticated/_dashboard/projects/index.tsx [hunk 3]
```diff
@@ -1,10 +1,236 @@
...
+              <Typography
+                variant="body1"
+                sx={{
+                  fontWeight: "bold",
+                  backgroundColor: BCDesignTokens.themeGold10,
+                }}
+              >
+                Past Submissions
+              </Typography>
+              <CardInnerBox sx={{ height: "100%" }}>
```
Comment (jadmnsaad) [inline submit-web/src/routes/_authenticated/_dashboard/projects/index.tsx]: just take out everything under Active submissions into a separate component and everything under Past Submissions into a separate component

File: submit-web/src/routes/_authenticated/_dashboard.tsx [hunk 4]
```diff
@@ -15,7 +15,7 @@ function DashboardLayout() {
   return (
     <div>
       <BreadcrumbNav />
-      <Box height={"calc(100vh - 88px)"} flexDirection={"row"} display={"flex"}>
+      <Box flexDirection={"row"} display={"flex"} sx={{ mb: 2 }}>
```
Comment (jadmnsaad) [inline submit-web/src/routes/_authenticated/_dashboard.tsx]: why the mb:2?

File: submit-web/src/components/registration/addProjects/ProjectStatus.tsx [hunk 5]
```diff
@@ -18,7 +19,9 @@ export const ProjectStatus = ({ status }: ProjectStatusProps) => {
           color={"#947BB6"}
         >
           <ModeStandbyIcon />
-          <Caption2 color={"#947BB6"}>Post-Decision</Caption2>
+          <Caption2 color={"#947BB6"} bold={bold}>
```
Comment (jadmsaadaot) [inline submit-web/src/components/registration/addProjects/ProjectStatus.tsx:22]: hard coded color?

File: submit-web/src/components/registration/addProjects/ProjectCard/constants.ts [hunk 6]
```diff
@@ -1,3 +1,5 @@
 export const PROJECT_STATUS = {
+  APPROVAL: "Approved",
+  IN_REVIEW: "In Review",
```
Comment (jadmnsaad) [inline submit-web/src/components/registration/addProjects/ProjectCard/constants.ts]: These are not related to the chip status on the right column. This is the work phase of the project
image

File: submit-web/src/components/Shared/ProjectStatusChip.tsx [hunk 7]
```diff
@@ -0,0 +1,39 @@
+import { Chip } from "@mui/material";
+import { PROJECT_STATUS } from "../registration/addProjects/ProjectCard/constants";
+import { BCDesignTokens } from "epic.theme";
+
+export default function ProjectStatusChip({ status }: { status: string }) {
```
Comment (jadmnsaad) [inline submit-web/src/components/Shared/ProjectStatusChip.tsx]: This is more like submission package status, it's not the project status. We do not manage the projects in submit, we manage submitting management plans for those projects

File: submit-web/src/routes/_authenticated/_dashboard/projects/index.tsx (hunk 1 above)
Comment (djnunez-aot) [reply submit-web/src/routes/_authenticated/_dashboard/projects/index.tsx]: nvm i found a fix

File: submit-web/src/routes/_authenticated/_dashboard.tsx (hunk 4 above)
Comment (djnunez-aot) [reply submit-web/src/routes/_authenticated/_dashboard.tsx]: removed it

**Review by jadmnsaad — APPROVED** [review]
